src/react/utils.js

```diff
@@ -704,3 +706,39 @@ export function getComponentName(
   }
   return "Unknown";
 }
+
+export function convertConfigObjectToReactComponentTreeConfig(
+  realm: Realm,
+  config: ObjectValue | UndefinedValue
+): ReactComponentTreeConfig {
+  // defaults
+  let serverSideRenderOnly = false;
```

Comment:
Should we call it firstRenderOnly? It won’t execute on the server and we’ll probably want different output modes for server rendering and client first render so we’ll need to separate them.

Reply:
Yeah, I don't mind, I'll change it in the PR that actually does this logic.